src/js/select2/data/tokenizer.js

```diff
@@ -70,6 +70,12 @@ define([
   Tokenizer.prototype.tokenizer = function (_, params, options, callback) {
     var separators = options.get('tokenSeparators') || [];
     var term = params.term;
+
+    var enclosureOptions = (options.get('tokenEnclosures') || ['\"', '\'']);
```

Comment:
It would be preferred to have this default to an empty array instead of defaulting to quotes. This would allow us to better maintain backwards compatibility when this is brought in.